browser/src/map/handler/Map.WOPI.js

```diff
@@ -462,7 +462,13 @@ L.Map.WOPI = L.Handler.extend({
 			msg.Values && msg.Values.id) {
 			this._map.uiManager.insertButton(msg.Values);
 			return;
-		} else if (msg.MessageId === 'Send_UNO_Command' && msg.Values && msg.Values.Command) {
+		}
+		else if (msg.MessageId === 'Insert_ContextualButton' &&
```

Comment:
can we add example usage (button) in the post message demo page?
when you `make run` there is a link for that page, we have html for it

Reply:
added!

Reply:
thank you very much!